### PR TITLE
Fix deprecations in Virtual Repeat and reintegrate with paper-autocomplete

### PR DESCRIPTION
**Pull request by ibarrick**
Fixed the deprecations in Virtual Repeat and reintegrated with paper-autocomplete

### CODE REVIEW

File: addon/components/paper-virtual-repeat.js [hunk 1]
```diff
@@ -60,14 +60,28 @@ export default VirtualEachComponent.extend({
...
       }
     }
-    this.set('height', this.get('horizontal') ? this.$()[0].clientWidth : this.$()[0].clientHeight);
+    run.schedule('actions', () => {
+      this.set('height', this.get('horizontal') ? this.$()[0].clientWidth : this.$()[0].clientHeight);
+    });
 
   },
+  offsetterStyle: computed('horizontal', '_marginTop', {
+    get() {
```
Comment (miguelcobain) [inline addon/components/paper-virtual-repeat.js]: We're using `computed('key', function() {` style for get computeds in the project.

**Comment by ibarrick** [comment]
switched them over

**Comment by ibarrick** [comment]
@miguelcobain Is there anything I can do to get this merged?

**Comment by ibarrick** [comment]
Closing as it would likely conflict with #487

**Pull request closed without merging**